content/en/docs/tasks/network/customize-hosts-file-for-pods.md

```diff
@@ -118,9 +118,9 @@ with the additional entries specified at the bottom.
 ## Why does the kubelet manage the hosts file? {#why-does-kubelet-manage-the-hosts-file}
 
 The kubelet [manages](https://github.com/kubernetes/kubernetes/issues/14633) the
```

Comment:
```suggestion
The kubelet manages the
```

Reply:
Sorry, I missed that PR had reference to Sucker. Thanks for pointing out!